tests/ai/src/index.ts

```diff
@@ -0,0 +1,166 @@
+import { App } from '@microsoft/teams.apps';
+import { DevtoolsPlugin } from '@microsoft/teams.dev';
+// :snippet-start: ai-imports
+import { ChatPrompt } from '@microsoft/teams.ai';
+import { OpenAIChatModel } from '@microsoft/teams.openai';
+// :snippet-end:
+import { MessageActivity } from '@microsoft/teams.api';
+import {
+  feedbackLoopCommand,
+  pokemonCommand,
+  ragCommand,
+  streamCommand,
+  weatherCommand,
+} from './commands';
+import { storedFeedbackByMessageId } from './feedback';
+import { handleDocumentationSearch } from './simple-rag';
+import { handleStatefulConversation } from './stateful-prompts';
+
+const app = new App({
+  plugins: [new DevtoolsPlugin()],
+});
+
+const model = new OpenAIChatModel({
+  apiKey: process.env.AZURE_OPENAI_API_KEY || process.env.OPENAI_API_KEY,
+  endpoint: process.env.AZURE_OPENAI_ENDPOINT,
+  apiVersion: process.env.AZURE_OPENAI_API_VERSION,
+  model: process.env.AZURE_OPENAI_MODEL_DEPLOYMENT_NAME!,
+});
+
+// Handle "hi" message
+// :snippet-start: simple-chat
+app.on('message', async ({ send, activity, next }) => {
+  // :remove-start:
+  if (activity.text.toLowerCase() !== 'hi') {
+    await next();
+    return;
+  }
+  // :remove-end:
+  const model = new OpenAIChatModel({
+    apiKey: process.env.AZURE_OPENAI_API_KEY || process.env.OPENAI_API_KEY,
+    endpoint: process.env.AZURE_OPENAI_ENDPOINT,
+    apiVersion: process.env.AZURE_OPENAI_API_VERSION,
+    model: process.env.AZURE_OPENAI_MODEL_DEPLOYMENT_NAME!,
+  });
+
+  const prompt = new ChatPrompt({
+    instructions: 'You are a friendly assistant who talks like a pirate',
+    model,
+  });
+
+  const response = await prompt.send(activity.text);
+  if (response.content) {
+    const activity = new MessageActivity(response.content).addAiGenerated();
+    await send(activity);
+    // Ahoy, matey! 🏴‍☠️ How be ye doin' this fine day on th' high seas? What can this ol’ salty sea dog help ye with? 🚢☠️
```

Comment:
delete? Or is this a comment indicating what the response will be?

Reply:
Yup indicating